examples/create_simple_sbom/main.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/anchore/syft/internal/licenses"
```

Comment:
this doesn't work as is since the example is suggesting for external API users to import from `syft/internal` which isn't possible

Reply:
I think the easiest thing to do would be to export the existing `setupContext`